src/components/table/table.tsx

```diff
@@ -279,6 +291,24 @@ export class Table {
         this.init();
     }
 
+    @Watch('aggregates')
+    protected updateAggregates(
+        newAggregates: ColumnAggregate[],
+        oldAggregates: ColumnAggregate[]
+    ) {
+        if (isEqual(newAggregates, oldAggregates)) {
```

Comment:
I think it's good to add a check for this.tabulator here, to be consistent with other functions

Reply:
@anderssonjohan Fixed!